dashboard/app/mailers/parent_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class ParentMailer < ActionMailer::Base
+  default from: 'noreply@code.org'
```

Comment:
The email text says they can reply to the email, but does `noreply` go anywhere?